__tests__/app.test.js

```diff
@@ -86,6 +86,63 @@ describe('Test GET api/articles  endpoint', () => {
         })
       })
   })
+  test('Test queries sort_by and order in ascending order', () => {
+    return request(app)
+      .get('/api/articles?sort_by=topic&order=asc')
```

Comment:
It might have been nicer to test sort_by and order separately before testing them together